packages/matchbox/src/components/Banner/Banner.tsx

```diff
@@ -57,17 +61,43 @@ const StyledDismiss = styled(Box)`
   ${dismissColor}
 `;
 
-const Banner = React.forwardRef(function Banner(props, userRef) {
+type ActionProps = React.ComponentProps<typeof Button> & {
+  content: string;
+};
+
+interface BannerProps extends React.ComponentPropsWithoutRef<'div'>, MarginProps {
+  children?: React.ReactNode;
+  'data-id'?: string;
```

Comment:
does data-id need to be specified since we have `React.ComponentPropsWithoutRef<'div'>`? Same with `id`